sklearn/metrics/_classification.py

```diff
@@ -771,6 +771,11 @@ def jaccard_score(
     >>> jaccard_score(y_true[0], y_pred[0])
     0.6666...
 
+    In the 2D comparison case (e.g. image similarity):
+
+    >>> jaccard_score(y_true.flatten(), y_pred.flatten())
```

Comment:
This is the same as `average="micro"`.

```suggestion
    >>> jaccard_score(y_true, y_pred, average='micro')
```

(We do not need to `flatten` which makes a copy)

Reply:
Corrected in commit below